Geometry/CommonDetUnit/interface/GluedGeomDet.h

```diff
@@ -0,0 +1,28 @@
+#ifndef Geometry_TrackerGeometryBuilder_GluedGeomDet_H
+#define Geometry_TrackerGeometryBuilder_GluedGeomDet_H
+
+#include "Geometry/CommonDetUnit/interface/TrackerGeomDet.h"
+#include "DataFormats/DetId/interface/DetId.h"
```

Comment:
The BuildFile.xml file for Geometry/CommonDetUnit should also be changed to depend upon DataFormats/DetId.